### PR TITLE
Fix media popup on Activity page

### PR DESCRIPTION
**Pull request by malavasita**

### CODE REVIEW

File: app/assets/js/rtMedia.js [hunk 1]
```diff
@@ -207,17 +207,22 @@ jQuery( 'document' ).ready( function( $ ) {
 
         // When Ajax completed attach media uploader to new activity, applay popup and attach media to comment uploader.
         jQuery(document).ajaxComplete( function(event, xhr, settings) {
-            if ( bp_template_pack && bp_template_pack === 'nouveau' ) {  
+			if ( bp_template_pack && bp_template_pack !== 'legacy' ) {
```
Comment (milindmore22) [inline app/assets/js/rtMedia.js]: fix alignment

**Comment by malavasita** [comment]
Alignment fixed @milindmore22 :+1:

File: app/assets/js/rtMedia.js [hunk 2]
```diff
@@ -204,20 +204,24 @@ function rtmedia_init_action_dropdown( parent ) {
...
-                        rtmedia_activity_stream_comment_media();
-                    } , 1000 );
-                }
-            }
-        } );
+	// When Ajax completed attach media uploader to new activity, applay popup and attach media to comment uploader.
+	jQuery(document).ajaxComplete( function(event, xhr, settings) {
+		if ( bp_template_pack && bp_template_pack !== 'legacy' ) {
+			var get_action = get_parameter( 'action', settings.data );
+			if ( ( 'activity_filter' === get_action ||  'post_update' === get_action || 'get_single_activity_content' === get_action || 'activity_get_older_updates' === get_action ) && typeof rtmedia_masonry_layout != 'undefined' && rtmedia_masonry_layout == 'true' && typeof rtmedia_masonry_layout_activity != 'undefined' && rtmedia_masonry_layout_activity == 'true' ) {
```
Comment (milindmore22) [inline app/assets/js/rtMedia.js]: strict comparison

**Comment by juhi123** [comment]
Looks good 👍

**Pull request merged**